decred/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "decred"
-version = "0.0.1"
+version = "0.1"
```

Comment:
Shouldn't this be `0.1.0`?

https://semver.org/

Reply:
Uhm. Yes, it should.